native_client/README.md

```diff
@@ -23,7 +56,7 @@ Before building the DeepSpeech client libraries, you will need to prepare your e
 Then you can build the Tensorflow and DeepSpeech libraries.
 
 ```
-bazel build -c opt --copt=-march=native --copt=-mtune=native --copt=-O3 //tensorflow:libtensorflow_cc.so //native_client:*
+bazel build -c opt --copt=-O3 //tensorflow:libtensorflow_cc.so //native_client:deepspeech //native_client:deepspeech_utils //native_client:ctc_decoder_with_kenlm //native_client:generate_trie
```

Comment:
We miss the configure step, right? Also, we should document native optimizations for people building themselves:
`--copt=-march=native` :)

Reply:
The configure step is in the TensorFlow docs. We explicitly tell people to follow the docs until the configure step.

TensorFlow uses `-march=native` by default in opt builds, so that's just command line spam.